### DISCUSSION TITLE
v.4.3.1

### DISCUSSION THREAD

**Discussion by DanRobo76** [DanRobo76/VMC-HELTY-FLOW#11] [Generale]
Codice YAML Release 30/06/2024

Elenco VMC installate; Pannello "Informazioni" -> "17 Elenco VMC Installate";
Corretto l'errore Time-out;
Migliorato il codice e le Logiche di Funzionamento;

Scheda Interfaccia Release 15/06/2024
Automazione Release 4.3.1 del 27/06/2024
Configurazione 30/06/2024

CLONA VMC HELTY FLOW PLUS Release V2.0.0
 - permette ora di cancellare la cartella digitata e il suo contenuto correlato

Codice Bash Release 11/11/2023 [Invariato]

<hr /><em>This discussion was created from the release <a href='https://github.com/DanRobo76/VMC-HELTY-FLOW/releases/tag/v.4.3.1'>v.4.3.1</a>.</em>

**Comment by pet1972s** [comment]
Buongiorno,
Vorrei installarlo sul mio Home Assistant, ma da PC - sulla stessa rete - non mi permette l'accesso alle cartelle del Raspberry PI.
Mi puoi consigliare? Oppure l'unico metodo è estrarre la MicroSD ed accedervi direttamente?
Grazie mille

**Comment by DanRobo76** [comment]
Ciao,
installa l'Add-On Samba-Share: Impostazioni->Componenti
Aggiuntivi->raccolta di componente aggiuntivi -> nel campo search inserisci
samba share

Per maggior dettagli puoi cercare su Youtube

Ciao!

Il giorno ven 5 lug 2024 alle ore 10:09 pet1972s ***@***.***>
ha scritto:

**Comment by pet1972s** [comment]
Grazie mille della risposta, sono riuscito ad installarlo.
Approfitto della gentilezza per una domanda: vedo tutte le temperature dei
sensori (sia VMC che meteo) con valori negativi.
E' normale? Oppure devo azzerare gli offset?

[image: image.png]

Il giorno ven 5 lug 2024 alle ore 14:07 DanRobo76 ***@***.***>
ha scritto:

**Comment by pet1972s** [comment]
Ho risolto con gli offset, anche per correggere il sensore umidità, che è
pessimo.
Complimenti per l'ottimo lavoro.
Grazie mille

Steven

Il sab 6 lug 2024, 18:41 Steven Pedreschi ***@***.***> ha
scritto:

**Comment by DanRobo76** [comment]
Si  azzera gli offset , poi vanno regolati se necessario

Il sab 6 lug 2024, 20:12 pet1972s ***@***.***> ha scritto:
